src/features/events_list/components/BBoxFilterToggle/BBoxFilterToggle.tsx

```diff
@@ -2,19 +2,18 @@ import { Button } from '@konturio/ui-kit';
 import { CloseFilled16, Update16 } from '@konturio/default-icons';
 import { useAtom } from '@reatom/react-v2';
 import { i18n } from '~core/localization';
-import { featureFlagsAtom, FeatureFlag } from '~core/shared_state';
 import { eventListFilters } from '../../atoms/eventListFilters';
 import s from './BBoxFilterToggle.module.css';
 
+// FeatureFlag: EVENTS_LIST__BBOX_FILTER
```

Comment:
Do we need to keep the commented line here?

Reply:
But it's a comment, not commented out code)

Reply:
My bad :)